.gitlab-ci.yml

```diff
@@ -738,7 +738,11 @@ library:ci-coqtail:
   extends: .ci-template
 
 library:ci-coquelicot:
-  extends: .ci-template
+  stage: stage-3
+  extends: .ci-template-flambda
+  needs:
+  - build:edge+flambda
+  - library:ci-mathcomp
```

Comment:
Because the template has `dependencies` you need to duplicate them here too.
I think this is why the coquelicot job rebuilt mathcomp.

Reply:
I hope I fixed it